<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorProgressDialog</class>
 <widget class="QDialog" name="EditorProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Action In Progress...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="m_statusGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Status</string>
   </property>
   <widget class="QLabel" name="m_status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>621</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="m_progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="m_cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
